blog/modules/ROOT/pages/from-gitbook-to-antora.adoc

```diff
@@ -132,13 +138,20 @@ content:
 Since we are working with a local repository we are using the value dot (`.`). +
 The path will be resolved relative to the location of the playbook file.
 
-The `start_path` value points to our documentation component named `book`.
+The `start_path` value points to our documentation component located in the `book` directory (and also named `book`).
+
+[NOTE]
+====
+For ease of migration, this guide is recommending a particular very simple, but restrictive, layout in a single git repository.
+For a local repository, the -url will be the path from the playbook to the git root.
+The start_path is the path from the git root to the `antora.yml` component descriptor.
```

Comment:
We should use monospaced font for `start_path`

Reply:
fixed